dash/testing/plugin.py

```diff
@@ -30,7 +30,15 @@ def pytest_addoption(parser):
     )
 
     dash.addoption(
-        "--headless", action="store_true", help="Run tests in headless mode"
+        "--headless",
+        action="store_true",
+        help="set this flag to run in headless mode",
+    )
+
+    dash.addoption(
+        "--percyfinalize",
+        action="store_false",
+        help="set this flag to control percy finalize at CI level",
```

Comment:
would this make more sense to be called `--percynofinalize`? `"store_false"` just seems inherently confusing...

Reply:
`--nopercyfinalize`?

Reply:
> `--nopercyfinalize`

even better!